docs/intro-to-oss/how-to-contribute-to-open-source.md

```diff
@@ -426,6 +445,8 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
     :::
 
```

Comment:
This looks good. Just one more change. 

Add a heading and text below.

``
### Completed PR Template

Here's what a completed Pull Request Template for the guestbook looks like. Make sure yours looks like this, updated with your unique information.

``

Reply:
Updated my pr with the requested changes

![pr template](https://github.com/user-attachments/assets/fd8823c0-ea51-425d-affa-e0415897a2bb)